h2o-assembly/build.gradle

```diff
@@ -7,6 +7,7 @@ dependencies {
   compile project(":h2o-app")
   compile project(":h2o-persist-s3")
   compile project(":h2o-persist-hdfs")
+  compile project(':h2o-scala_2.10')
```

Comment:
@mmalohlava what do you think about the solution in the comment above ? I know that this wasn't nice solution - it was just temporary to make it work in order to perform tests